CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - `mode` property from `focusZone` configuration in accessibility behaviors is no longer supported - the focus zone will always be in embed mode @layershifter ([#2265](https://github.com/microsoft/fluent-ui-react/pull/2265))
 - `FocusZoneMode` and `FOCUSZONE_WRAP_ATTRIBUTE` are no longer exported @layershifter ([#2265](https://github.com/microsoft/fluent-ui-react/pull/2265))
 - Changed `avatarBorderWidth` and `statusBorderWidth` avatar variables types from number to string and updated styles in Teams theme @mnajdova ([#2238](https://github.com/microsoft/fluent-ui-react/pull/2238))
+- Add logic for mounting/removing elements when they are shown/hidden using the `Animation` component @mnajdova ([#2115](https://github.com/microsoft/fluent-ui-react/pull/2115))
```

Comment:
Is it a breaking change?

Reply:
It is breaking, but mostly because the Animation component is not rendering div anymore, so maybe that should be mention here too.. Will add it